site-src/geps/gep-1282.md

```diff
@@ -90,6 +97,7 @@ We’re looking to add specific, structured extension points somewhere in the re
 Those specific, structured extension points need to be in a place where they can be owned by the person who owns the backend, since that could be different to the person who owns the Route. That is, whatever we choose should be something that extends a thing that is referred to by a `backendRef` in a Route, not inside the Route.
 
 The initial list includes, but is not limited to:
+
 * TLS information for connection from the Gateway to the backend service. Note that this doesn’t include any information used for service mesh encryption, just what a Gateway’s proxy would need to be able to connect to the backend service.
 * Websocket protocol information for the backend service.
```

Comment:
I don't think mixing TLS info with protocol info is ideal. TLS is pretty advanced and scary for many users.
Also in many cases this is negotiated at protocol level, not config ( ALPN, etc)